users/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+class User(models.Model):
+    nickname     = models.CharField(max_length=30, unique=True)
+    email        = models.EmailField(max_length=100, unique=True)
+    password     = models.CharField(max_length=200)
+    phone_number = models.CharField(max_length=15, unique=True)
+    gender       = models.CharField(max_length=5)
+    birth        = models.CharField(max_length=12)
+    product      = models.ManyToManyField('products.Product', through='Like')
```

Comment:
```suggestion
    like      = models.ManyToManyField('products.Product', through='Like')
```
변수명을 다음과 같이 바꾸는 것이 조금 더 명확 할 것 같습니다.

Reply:
중간 테이블이름으로 바꾸는 것에 대해 혼돈이 있어 구두로 질문 드리겠습니다.

Reply:
ManyToManyField는 DB 생성되는 필드가 아니므로 product라고 할 시 user.product가 의미하는것이 like인지 모릅니다.
그래서 알 수 있는 like로 바꾸는 것이 좋다고 구두로 설명해주셨습니다.

감사합니다.